Meta/CMake/lagom_options.cmake

```diff
@@ -9,3 +9,4 @@ serenity_option(ENABLE_MEMORY_SANITIZER OFF CACHE BOOL "Enable memory sanitizer
 serenity_option(ENABLE_FUZZER_SANITIZER OFF CACHE BOOL "Enable fuzzer sanitizer testing in clang")
 serenity_option(BUILD_LAGOM OFF CACHE BOOL "Build parts of the system targeting the host OS for fuzzing/testing")
 serenity_option(ENABLE_LAGOM_CCACHE ON CACHE BOOL "Enable ccache for Lagom builds")
+serenity_option(ENABLE_LAGOM_FUZZER OFF CACHE BOOL "Build Lagom fuzzing targets")
```

Comment:
All of the fuzzer flags are a bit confusing in my experience, I think this has the potential of just making this even more confusing. Is there a way to fix the documentation of the flags / or names so that it's less opaque and more obvious? 

Variants I know of:
- Fuzzers with OSS Fuzz
- Fuzzers with libFuzz
- (This) Fuzzers with the Lagom Shim

Reply:
Those three should be all the ones that currently exist. However, the third is intended to be a general-purpose "enable building the fuzzers flag" and is automatically included when we either want to build fuzzers for OSS-Fuzz or fuzzers for libFuzz. Sadly, `ENABLE_LAGOM_FUZZER` was the best name that I could come up with so far.

Do we need to keep compatibility with anything external? Or are all the CMake flags self-contained within the repository? If so, we could rename the existing flags (`ENABLE_FUZZERS_OSSFUZZ` and `ENABLE_FUZZERS_LIBFUZZ`) and the new flag could be `ENABLE_FUZZERS`. In my opinion, that should make it quite clear which flags depend on each other.

Reply:
I think as long as `Meta/Lagom/BuildFuzzers.sh` is updated, then everything should work. 
The OSS Fuzz docker container just calls into there: https://github.com/google/oss-fuzz/blob/505b4a4ec537231078e7273bb0bd6ed5de267097/projects/serenity/